### DISCUSSION TITLE
.net MAUI should be .NETUI

### DISCUSSION THREAD

**Discussion by hexbinoct** [dotnet/maui#3883] [Ideas]
just had a thought, Apple's SwiftUI is a beautiful name for their UI tech for ALL Apple devices, just like what Microsoft is doing with .Net MAUI. If the name changes to .NETUI, it will be so simple and clear and distinguish it from anything that came before it from Microsoft.

**Comment by GeraudFabien** [comment]
A lot of people agree about the fact MAUI is a really bad name. Not necessarily because it's cute but because it's already taken.
https://github.com/dotnet/maui/issues/35
And it also have meaning in [Māori](https://en.wikipedia.org/wiki/M%C4%81ui_(M%C4%81ori_mythology)).

There was already another discussion on the topic and somethink close to NetUI was already propose :
https://github.com/dotnet/maui/discussions/41#discussioncomment-31965

> beautiful

It's a personal opinion. MAUI is a good name too. If it wasn't already taken by a similar project there would be no problem. I personally don't like SwiftUI or NetUI for exemple. To me NetUI would be SkiaSharp.

**Comment by chris-ux** [comment]
I don't care one way or the other about the name. My **only** issue with name is the SEO of travel/vacation companies websites and personal travel blogs getting in the way of search when I am looking for Maui stuff without adding .net or c# to the search.

**Reply by hartez** [reply]
That's one thing I really miss about using a made-up word like Xamarin - no search conflicts.

**Comment by davidbuckleyni** [comment]
Not really net implies that it was made for the web aa Maui embrasses desktop and mobile I do not see how that name fits.
